packages/dds/sequence/src/sequence.ts

```diff
@@ -452,20 +452,13 @@ export abstract class SharedSegmentSequence<T extends ISegment>
             entries,
         };
 
-        return tree;
```

Comment:
Feels a bit inconsistent that in other places with two blobs you converted code to use native summary functionality, while here you left ITree -> Summary conversion

Reply:
For anything complicated (like the tree entry above), I used convertToSummaryTreeWithStats rather than converting all the code to create summaries directly. I'll defer that fix-up to #8652 for now.